frontend/src/bundles/resume/components/delete-resume-button/delete-resume-button.tsx

```diff
@@ -0,0 +1,34 @@
+import { useCallback, useState } from 'react';
+
+import deleteIcon from '~/assets/img/delete-icon.svg';
+import { BaseButton } from '~/bundles/common/components/components';
+
+import { DeleteResumeModal } from '../delete-resume-modal.tsx/delete-resume-modal';
+import styles from './styles.module.scss';
+
+const DeleteResumeButton: React.FC = () => {
+    const [isModalShown, setIsModalShown] = useState(false);
+
+    const handleShowModal = useCallback(() => {
+        setIsModalShown(true);
+    }, []);
+
+    const handleCloseModal = useCallback(() => {
+        setIsModalShown(false);
+    }, []);
+
+    return (
+        <div className={styles.delete__button__wrapper}>
+            <BaseButton
+                onClick={handleShowModal}
+                className={styles.delete__button}
+            >
+                <img src={deleteIcon} alt="delete icon" />
```

Comment:
It's not the best way of using icons. There should be something like `Icon` component for it